nativescript-angular/router/page-router-outlet.ts

```diff
@@ -125,14 +127,16 @@ export class PageRouterOutlet implements OnDestroy { // tslint:disable-line:dire
 
     get component(): Object {
         if (!this.activated) {
-            throw new Error("Outlet is not activated");
+            log("Outlet is not activated");
+            return;
         }
 
         return this.activated.instance;
     }
     get activatedRoute(): ActivatedRoute {
         if (!this.activated) {
-            throw new Error("Outlet is not activated");
+            log("Outlet is not activated");
```

Comment:
@vakrilov worth noticing that obviously this logic mirrors what Angular framework does for `router-outlet`: https://github.com/angular/angular/blob/master/packages/router/src/directives/router_outlet.ts#L76-L83 however again argument here is {N} is not a web app and therefore cannot withstand a `throw` in userland due to detrimental effect app crash has in mobile apps vs. web apps.